### DISCUSSION TITLE
Custom URL?

### DISCUSSION THREAD

**Discussion by brianushman** [udayvunnam/xng-breadcrumb#88] [Q&A] [answered]
Hello, I'd like to know how I can override the URL of a breadcrumb and change it.

Why want to do this?
It's because our "Home" link needs to stay an empty route in Angular for proper routing.  However, I'd like the "Home" breadcrumb itself to point to our legacy application which is a totally different framework. Is this possible to override the URL or set a custom value?  

I've tried using the "routeInterceptor" on the breadcrumb however it appends my url to the existing one set through the route.  Not sure how to specify my own full URL.

Thanks!

**Comment by udayvunnam** [comment, marked as answer]
@brianushman you can do this easily. Refer to the advanced guide - https://github.com/udayvunnam/xng-breadcrumb/blob/main/docs/advanced-patterns.md#intercept-the-routing-via-breadcrumb-navigation---routeinterceptor
